src/components/Learn-Components/Pagination/index.js

```diff
@@ -51,14 +51,14 @@ const Pagination = ({ TOCData, chapterData, location, showQuizModal }) => {
           </div>
         </div>
       </PaginationWrapper>) 
-      : null
-      // : (
-      //       <QuizWrapper onClick={showQuizModal}>
-      //           <Button secondary title="Take Quiz"
-      //             external={false} 
-      //           />
-      //         </QuizWrapper>
-      //   )
+      // : null
+      : (
+        <QuizWrapper onClick={showQuizModal}>
+          <Button secondary title="Take Quiz"
+            external={false} 
+          />
+        </QuizWrapper>
+      )
```

Comment:
The fix is working, let's get it back commented.

Reply:
done @Nikhil-Ladha